### PR TITLE
Don't set node-os-distro flag on kops jobs

### PR DESCRIPTION
**Pull request by upodroid**
/cc @dims 

I'm going to code this kops's ginkgo tester

### CODE REVIEW

File: config/jobs/kubernetes/kops/build_jobs.py [hunk 1]
```diff
@@ -869,7 +869,6 @@ def generate_misc():
                    ],
                    skip_regex=r'\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[KubeUp\]', # pylint: disable=line-too-long
                    test_timeout_minutes=60,
-                   test_args="--master-os-distro=gci --node-os-distro=gci",
```
Comment (aojea) [inline config/jobs/kubernetes/kops/build_jobs.py:872]: the name of the job is `ci-kubernetes-e2e-cos-gce-canary` , is ok to change the os distro flags?

Comment (aojea) [reply config/jobs/kubernetes/kops/build_jobs.py:872]: also it sets an image `"--image=cos-cloud/cos-105-17412-156-49",`

Comment (upodroid) [reply config/jobs/kubernetes/kops/build_jobs.py:872]: I'm going to adjust the kops ginkgo tester to automatically insert these flags.

One important flag that is missing is -num-nodes which is causing weird skips everywhere.

This file 

https://github.com/kubernetes/kops/blob/master/tests/e2e/pkg/tester/tester.go

Comment (aojea) [reply config/jobs/kubernetes/kops/build_jobs.py:872]: 👍  it's ok to for the external tooling to handle these things, but we should avoid bringing this complexity into the kubernetes/kubernetes tests, the tests should only implement the test functionality, and the users should decide which ones to run or skip

**Pull request merged**